.github/workflows/rebuild-released-images.yaml

```diff
@@ -63,22 +64,13 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v4
         with:
-          ref: ${{ matrix.version }}
+          ref: "v${{ matrix.version }}"
           submodules: true
           fetch-depth: 0
-      - name: "Set up QEMU"
-        uses: docker/setup-qemu-action@v3
-        with:
-          platforms: ${{ fromJSON(needs.read-versions.outputs.matrix).platforms }}
-      - name: "Set up Docker Buildx"
-        uses: docker/setup-buildx-action@v3
-        with:
-          platforms: ${{ fromJSON(needs.read-versions.outputs.matrix).platforms }}
-      - name: Login to docker registry
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Upgrade build Makefile & Dockerfile
```

Comment:
Was this added to be able to test it?

Reply:
Not really, this is needed because of the new build code. As the rebuild uses the old released commit, it needs to patch the `Makefile` and add the new `fast.Dockerfile` to be able to build properly using the new way. Once the last version has moved to the new way we would be able to remove this.

Reply:
Not really. This is required as the daily build executes in old code which does not have the newer Makefile and fast.Dockerfile that is used by this new build process.